### PR TITLE
Update mongo-express for v1.0.2

### PR DESCRIPTION
**Pull request by BlackthornYugen**
## Related pull requests:
https://github.com/mongo-express/mongo-express-docker/pull/107
https://github.com/mongo-express/mongo-express-docker/pull/108

## Full release notes available here:
[v1.0.1 - Convert to ESM](https://github.com/mongo-express/mongo-express/releases/tag/v1.0.1)
[v1.0.2](https://github.com/mongo-express/mongo-express/releases/tag/v1.0.2)

### CODE REVIEW

File: library/mongo-express
Comment (yosifkit) [inline library/mongo-express:5]: I think it is this empty line that is tripping it up. With it, it considers just the `Maintainers:` above as the "global" section and then the `GitRepo` + `GitCommit` just below as a new item with no `Tags` (which it then complains about missing tags).

Comment (BlackthornYugen) [reply library/mongo-express]: Oh right -- let me try without that.

Comment (BlackthornYugen) [reply library/mongo-express]: Now it doesn't like my tags now. Says there isn't a ":" on that line? 

image

Comment (BlackthornYugen) [reply library/mongo-express]: Oh, that's exactly what's wrong. I'll double check the generate script. Fix it manually for now.

**Comment by BlackthornYugen** [comment]
Current failure is probably because we haven't merged the upstream changes in https://github.com/mongo-express/mongo-express-docker/pull/108 yet.

**Comment by BlackthornYugen** [comment]
Looks like we can't support alpine 3.17 anymore.

image

**Comment by tianon** [comment]
Was going to make a few review comments, but figured they'd be easier to digest as a PR: https://github.com/mongo-express/mongo-express-docker/pull/109 :+1: :heart:

**Comment by LaurentGoderre** [comment]
We need to update this after https://github.com/mongo-express/mongo-express-docker/pull/101 lands

**Review by LaurentGoderre — CHANGES_REQUESTED** [review]
Need to get the entrypoint changes in

**Comment by tianon** [comment]
I think I must be mis-reading https://github.com/mongo-express/mongo-express-docker/pull/101 -- it seems like that's just re-enabling the "wait for MongoDB at startup" logic?  Not sure why that's blocking :sweat_smile:

**Comment by BlackthornYugen** [comment]
We are missing the latest tag because we had it on alpine 1.17. I'll update to add alpine 1.19 and move the latest tag to 1.18.

**Review by LaurentGoderre — APPROVED** [review]

**Review by tianon — APPROVED** [review]

**Comment by tianon** [comment]
Not gonna block the merge on it (obviously :grin:), but just want to double check that it's intentional for `latest` to point to the `alpine3.18` variant and not `alpine3.19`? (easy fix in a follow-up if not :+1:)

**Comment by BlackthornYugen** [comment]
> Not gonna block the merge on it (obviously 😁), but just want to double check that it's intentional for `latest` to point to the `alpine3.18` variant and not `alpine3.19`? (easy fix in a follow-up if not 👍)

I did that on purpose. Wanted to keep changes small. 🙂

**Pull request merged**
